commands/host/install-dgi-playwright

```diff
@@ -1,19 +1,19 @@
 #!/bin/bash
 #ddev-generated
 
-mkdir -p test/playwright
-ddev mutagen sync
-
 # Adding the new dgi-playwright-tests installer type and path
-ddev composer require oomphinc/composer-installers-extender:^2.0
-ddev composer config --merge --json extra.installer-types '["dgi-playwright-tests"]'
-ddev composer config --merge --json extra.installer-paths.test/playwright '["type:dgi-playwright-tests"]'
+ddev composer require "oomphinc/composer-installers-extender:^2.0"
+ddev composer config --merge --json -- extra.installer-types '["dgi-playwright-core", "dgi-playwright-tests"]'
+ddev composer config --merge --json -- extra.installer-paths.test/playwright '["type:dgi-playwright-core"]'
+ddev composer config --merge --json -- 'extra.installer-paths.test/playwright-tests/{$name}' '["type:dgi-playwright-tests"]'
```

Comment:
Just had the thought, but: We should probably be placing a `.gitignore` that ignores these directories.